cmd/nerdctl/image/image_remove_test.go

```diff
@@ -351,3 +351,110 @@ func TestIssue3016(t *testing.T) {
 
 	testCase.Run(t)
 }
+
+func TestRemoveKubeWithKubeHideDupe(t *testing.T) {
```

Comment:
@apostasie Could you take a look?
Is this correct?
The test doesn't seem to involve with Kube

Reply:
Looking.

Reply:
@AkihiroSuda 

This is correct.

Repro locally:

```
# This creates the kind / kube cluster and configure what needs be
./hack/build-integration-kubernetes.sh
# Jump into the control plane to gain access to the created cluster
nerdctl exec -ti nerdctl-test-control-plane bash
```

This ^ is exactly what the CI is doing

Now that you are in the control plane, use nerdctl against the kind cluster:
```
nerdctl --namespace k8s.io images
nerdctl --namespace k8s.io pull busybox
nerdctl --namespace k8s.io images
```

^ now, `2919d0172f75` (in my case, on arm) appears as busybox and none/none

Reply:
> This is correct.

What I mean is: the test is actually doing what it says, and does interact with Kube, as nerdctl is interacting with the kind cluster and the k8s.io namespace